src/legacy/core_plugins/kibana/public/management/sections/index_patterns/index.js

```diff
@@ -103,7 +116,10 @@ uiModules.get('apps/management')
 
             return {
               id: id,
-              title: pattern.get('title'),
+              title:
+  <span>
+    {pattern.get('title')}{$scope.defaultIndex === id && (<EuiBadge style={{ marginLeft: '8px' }}>Default</EuiBadge>)}
```

Comment:
Would you mind creating a class in SCSS instead of an inline style? This way you can consume the EUI size variable, and avoid violating the CSP. @ryankeairns Can you provide guidance on how to do this?

Reply:
I had issues running this PR locally (I think due to ES version switching), so I can't see exactly the outcome here, but to answer the question you would do something like:

`&& (<EuiBadge className='indexPatternList__badge'>Default</EuiBadge>)`

...then add the `.indexPatternList__badge` class to the neighboring `index.scss` file:

```
.indexPatternList__badge {
  margin-right: $euiSizeS;
}
```

Reply:
++